plugins/woocommerce/changelog/50606-fix-49651-rest-remove-line-item-stock

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: fix
+
+Ensure that the orders REST endpoint behaves the same as the UI when updating an order to remove a line item.
```

Comment:
Would indicating we are specifically addressing a concern related to inventory handling be clearer/more precise (perhaps there are other ways in which this endpoint diverges from the UI, unrelated to inventory)?

```suggestion
In respect of restoring stock levels, ensure that the orders REST endpoint behaves the same as the UI when removing line items.
```

Reply:
I like it.